From 67a06f5b9afa132a187956d5f28b0470b84a0235 Mon Sep 17 00:00:00 2001
From: Patrick Totzke <patricktotzke@gmail.com>
Date: Sun, 16 Feb 2020 10:24:59 +0000
Subject: [PATCH] Update reference to envelope.body

which now should be body_txt as of e067ea2037c10224645f0cb7ab2e020772e612ac

fixes #1468
---
 alot/commands/globals.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/alot/commands/globals.py b/alot/commands/globals.py
index f9dc5812..aa1fd436 100644
--- a/alot/commands/globals.py
+++ b/alot/commands/globals.py
@@ -869,7 +869,7 @@ class ComposeCommand(Command):
                     mimetype = helper.guess_mimetype(sigcontent)
                     if mimetype.startswith('text'):
                         sigcontent = helper.try_decode(sigcontent)
-                        self.envelope.body += '\n' + sigcontent
+                        self.envelope.body_txt += '\n' + sigcontent
             else:
                 ui.notify('could not locate signature: %s' % sig,
                           priority='error')
-- 
2.24.1

